server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/ImapStepdefs.java

```diff
@@ -62,6 +62,18 @@ public void hasMessageInMailbox(String mailbox) throws Throwable {
         }
     }
 
+    @Then("^the message is \"([^\"]*)\" on \"([^\"]*)\"$")
```

Comment:
The message has IMAP flags ".\*" in mailbox ".\*"

You are not explicit enough...